src/test/storage/dictionary_segment_test.cpp

```diff
@@ -70,6 +70,127 @@ TEST_F(StorageDictionarySegmentTest, LowerUpperBound) {
   EXPECT_EQ(dict_segment->upper_bound(15), INVALID_VALUE_ID);
 }
 
-// TODO(student): You should add some more tests here (full coverage would be appreciated) and possibly in other files.
+TEST_F(StorageDictionarySegmentTest, AccessingElements) {
```

Comment:
We believe it may not be optimal to test multiple functions (``operator[]`` and ``get``) in the same test, as you lose visibility of where the error actually originated.